tsfresh/feature_extraction/extraction.py

```diff
@@ -174,6 +176,44 @@ def _extract_features_parallel_per_kind(kind_to_df_map, settings, column_id, col
     return result
 
 
+def _extract_features_serial(kind_to_df_map, settings, column_id, column_value):
```

Comment:
Two things here:
(1) profiling gets a lot harder (if not impossible) if you have multiprocessing. For example, tools like line_profiling etc do not work as expected
(2) If you do runtime studies, you do not want to be dependent on the CPUs scheduler and the OS, so you try to have the same setup all the time => singleprocessing should be better here